common/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Hyperion Grabber</string>
+    <string name="pref_title">Einstellungen</string>
+
+    <string name="enabled">Einschalten</string>
+    <string name="disabled">Ausschalten</string>
+
+    <!-- Network scanner -->
+    <string name="scanner_title">Hallo!</string>
+    <string name="scanner_description">Hyperion Grabber kann im Netzwerk nach Servern suchen, so dass Du nicht die IP-Adresse deines Servers manuell eingeben musst.</string>
+    <string name="scanner_start_scan_button">Suche starten</string>
+    <string name="scanner_scan_in_progress_text">Suche nach lokalen Hyperion Servers läuft %s</string>
+    <string name="scanner_scan_in_progress_button">Suche…</string>
+    <string name="scanner_no_results">Kein Server gefunden %s</string>
+    <string name="scanner_retry_button">Wiederholen</string>
+    <string name="scanner_manual_setup_button">Manuelle Einstellung</string>
+
+    <!-- Network Scanner result -->
+    <string name="scan_result_title">Und der Gewinner ist …</string>
+    <string name="scan_result_description">Jetzt müssten alle deine LEDs Lila aufleuchten %s</string>
+    <string name="scan_result_confirm_button_text">Ich liebe es!</string>
+    <string name="scan_result_alternative_color_button_text">Bitte kein Lila</string>
+
+    <!-- Connection settings -->
+    <string name="pref_group_connection">Verbindung</string>
+
+    <string name="pref_key_host">pref_key_host</string>
+    <string name="pref_title_host">IP Adresse</string>
+    <string name="pref_summary_host">Die IP Adresse auf dem dein Hyperion Server läuft</string>
+
+    <string name="pref_key_port">pref_key_port</string>
```

Comment:
pref_keys should not be added to translations. In the default language resource, these entries should be changed to eg.

`<string "translatable=false" name="pref_key_host">pref_key_host</string>`

http://tools.android.com/recent/non-translatablestrings

Reply:
Good catch, I wasn't aware of that translatable attribute.

I'm thinking of consolidating the strings to just the common directory so it will be easier to use tools like lokalise... any thoughts on that?

I'll send an invite for that to you as well!